Assets/Tracker.cs

```diff
@@ -73,28 +90,23 @@ public void Start ()
 			this.traceFormatter = new DefaultTraceFromat ();
 			break;
 		}
+		filePath = GeneratePath ();
 		switch (storageType) {
 		case "net":
-			storage = new NetStorage (this, host, trackingCode);
+			filePath += "Pending";
+			mainStorage = new NetStorage (this, host, trackingCode);
+			mainStorage.SetTracker (this);
+			localStorage = new LocalStorage (filePath);
+			localStorage.SetTracker (this);
 			break;
 		default:
-			String path = Application.persistentDataPath;
-#if UNITY_ANDROID
-			AndroidJavaObject env = new AndroidJavaObject("android.os.Environment");
-			AndroidJavaObject file = env.CallStatic<AndroidJavaObject>("getExternalStorageDirectory");
-			path = file.Call<String>("getAbsolutePath");
-#endif
-			if (!path.EndsWith ("/")) {
-				path += "/";
-			}
-			path += "traces";
-			if (debug) {
-				Debug.Log ("Storing traces in " + path);
-			}
-			storage = new LocalStorage (path);
+			mainStorage = new LocalStorage (filePath);
+			mainStorage.SetTracker (this);
+			localStorage = new LocalStorage(filePath+"Aux");
+			localStorage.SetTracker(this);
```

Comment:
Since they are identical, it seems unnecessary to initiate both storages. Maybe the default should always be the backup storage, and leave main storage uninitialized?